sqlmesh/core/audit/__init__.py

```diff
@@ -3,7 +3,14 @@
 from types import ModuleType
 
 from sqlmesh.core.audit import builtin
-from sqlmesh.core.audit.definition import Audit, AuditResult
+from sqlmesh.core.audit.definition import (
+    Audit,
+    AuditResult,
+    StandaloneAudit,
+    create_standalone_audit,
+)
+
+AuditType = t.Union[Audit, StandaloneAudit]
```

Comment:
I don't think a type contain `Type` in its name.

Reply:
Habit from that being the coding standard from where I worked.